server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
Yes, meta is a prefix in English. “Metadata” is also really the only kind of “meta” that is ever used in computer programming.

As such the name is clear and unambiguous. Note “md” not “mdesc” or “methods” or anything else more descriptive.

“m” is too short, because it's not descriptive enough. “meta” is a perfect choice here.